src/ebay-video/controls.tsx

```diff
@@ -9,7 +9,10 @@ export function customControls(onReport = () => {}): { Report } {
         constructor(parent, controls, text) {
             super(parent, controls)
 
-            appendChild(parent, <ReportButton onReport={onReport}>{text}</ReportButton>)
+            appendChild(parent, <ReportButton>{text}</ReportButton>, button => {
+                // have to listen to clicks this way (React onClick will not work):
+                this.eventManager.listen(button, 'click', () => onReport())
```

Comment:
Should we pass the event?

Reply:
do you mean `onReport`? It doesn't work in this weird google video player. that's why we have to use workarounds.

Reply:
Doesn't the eventManager calls the callback with an `event`?

```
 this.eventManager.listen(button, 'click', (EVENT) => onReport(event))
```

Reply:
fixed